hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/node/TestContainerPlacement.java

```diff
@@ -158,13 +159,13 @@ public void testContainerPlacementCapacity() throws IOException,
       assertEquals(remaining * nodeCount,
           (long) nodeManager.getStats().getRemaining().get());
 
-      xceiverClientManager= new XceiverClientManager(new OzoneConfiguration());
+      xceiverClientManager= new XceiverClientManager(conf);
 
       ContainerInfo container = containerManager
           .allocateContainer(
-          xceiverClientManager.getType(),
-          xceiverClientManager.getFactor(), "OZONE");
-      assertEquals(xceiverClientManager.getFactor().getNumber(),
+              SCMTestUtils.getReplicationType(conf),
+              SCMTestUtils.getReplicationFactor(conf), "OZONE");
```

Comment:
"OZONE" could be replaced by `OzoneConsts.OZONE` if it's not required to be in capital letters.
Otherwise, we can add a new constant in OzoneConsts.

Reply:
This magic constant is used in multiple tests, I would suggest to resolve this in a separate JIRA after this one get merged, I have created https://issues.apache.org/jira/browse/HDDS-2378 to track this, and I am open to post a patch for that as well.